core/src/banking_stage.rs

```diff
@@ -4411,7 +4418,7 @@ mod tests {
         );
         assert_eq!(
             expected_executions,
-            batched_transaction_details.costs.batched_execute_cost
```

Comment:
Can you expand this `test_accumulate_batched_transaction_costs` test to ensure builtin costs are accumulated correctly as well?

Reply:
done